lib/Configuration.js

```diff
@@ -122,16 +26,12 @@ const ENVIRONMENTS = {
   meteor: meteorEnvironment,
 };
 
-function checkForUnknownConfiguration(config: Object): Array<string> {
+function checkConfiguration(config: Object): Array<string> {
   const messages = [];
```

Comment:
This seems to be unused now

Reply:
Woah, good catch.

I sorted all the configuration options in another PR. This caused a merge conflict here, and I must've done somthing wrong when resolving it.

I have re-added it now.

Reply:
I think you responded to the wrong comment here. This array is still unused.

Reply:
Ah yes. Apologies. I responded to the wrong comment.

The unused variable 'messages' has now been removed.